src/protocols/secure_channel/CASESession.cpp

```diff
@@ -1642,6 +1642,15 @@ CHIP_ERROR CASESession::OnMessageReceived(ExchangeContext * ec, const PayloadHea
     Protocols::SecureChannel::MsgType msgType = static_cast<Protocols::SecureChannel::MsgType>(payloadHeader.GetMessageType());
     SuccessOrExit(err);
 
+#if CHIP_CONFIG_SLOW_CRYPTO
+    if (msgType == Protocols::SecureChannel::MsgType::CASE_Sigma1 || msgType == Protocols::SecureChannel::MsgType::CASE_Sigma2 ||
+        msgType == Protocols::SecureChannel::MsgType::CASE_Sigma2Resume ||
+        msgType == Protocols::SecureChannel::MsgType::CASE_Sigma3)
+    {
+        ReturnErrorOnFailure(mExchangeCtxt->SendStandaloneAckMessage());
```

Comment:
```suggestion
        SuccessOrExit(mExchangeCtxt->FlushAcks());
```
The right thing to call is FlushAcks. Because `SendStandaloneAckMessage` will send an ack even if an ack is not pending.

`SendStandaloneAckMessage` shouldn't even be public API... :(